utsw-bicf/nacculator
# 3/json-conversion

**What does this change do?** _Please be clear and concise._

Adds ability to convert a RedCap exported CSV file into the appropriate JSON format.

**Why was this change made?** _Including an issue number is sufficient. Otherwise, briefly explain the benefit of the change._

#13 , #9 
## Verification

_List the steps needed to make sure this thing works. Be precise._

1. Cd into the nacho folder
2. Run the following command (you may need to change the .csv file path): 
`clear && pip3 uninstall nacculator -y && pip3 install ../ && clear && python3 redcap2json.py --fileIn redcap-exported-file.csv`
3. The `.json` output file can be located in the same directory as `redcap2json.py`
- Verify the thing does this: ...
- Verify the thing does not do that: ...


## Affirmations

All of these should have a check by them. Any exception requires an explanation.
